episodes/config.json

```diff
@@ -0,0 +1,6 @@
+{
+  "token_file"    : "token.json",
+  "client_id"     : REPLACEME,
+  "client_secret" : REPLACEME,
```

Comment:
Let's not allow storing secrets in the same file as public data. Could be in `~/.local/share/nix-hour` or in a repo file which is then added to `.gitignore`